mimic/model/heat_objects.py

```diff
@@ -0,0 +1,232 @@
+"""
+Model objects for the Heat mimic.
+"""
+
+from characteristic import attributes, Attribute
+import json
+from random import randrange
+
+from mimic.model.behaviors import BehaviorRegistryCollection, EventDescription
+
+
+@attributes(['stack_name',
+             Attribute('stack_id',
+                       default_factory=lambda: Stack.generate_stack_id()),
+             Attribute('action', default_factory=lambda: Stack.CREATE),
+             Attribute('status', default_factory=lambda: Stack.COMPLETE),
+             Attribute('tags', default_factory=list)])
+class Stack(object):
+    """
+    A :obj:`Stack` is a representation of a Heat stack.
+    """
+
+    ACTIONS = (
+        CREATE, DELETE, UPDATE, ROLLBACK, SUSPEND, RESUME, ADOPT,
+        SNAPSHOT, CHECK, RESTORE
+    ) = (
+        'CREATE', 'DELETE', 'UPDATE', 'ROLLBACK', 'SUSPEND', 'RESUME', 'ADOPT',
+        'SNAPSHOT', 'CHECK', 'RESTORE'
+    )
+
+    STATUSES = (IN_PROGRESS, FAILED, COMPLETE
+                ) = ('IN_PROGRESS', 'FAILED', 'COMPLETE')
+
+    def json(self):
+        """
+        Returns the JSON representation of the stack.
+        """
+        return {
+            'stack_name': self.stack_name,
+            'stack_status': self.action + '_' + self.status,
+            'id': self.stack_id,
+            'tags': ','.join(self.tags)
```

Comment:
This list response seems different from what I see in http://docs.rackspace.com/orchestration/api/v1/orchestration-devguide/content/GET_stack_list__v1__tenant_id__stacks_Stack_Operations.html.